src/components/OrderPanel/BookingDatesForm/BookingDatesForm.js

```diff
@@ -508,27 +607,34 @@ export const BookingDatesFormComponent = props => {
           onFetchTimeSlots
         );
         const isDayBlocked = isDayBlockedFn({
-          allTimeSlots,
+          allTimeSlots: realTimeSlots,
           monthlyTimeSlots,
           isDaily: lineItemUnitType === LINE_ITEM_DAY,
           startDate,
           endDate,
           timeZone,
         });
         const isOutsideRange = isOutsideRangeFn(
-          allTimeSlots,
+          realTimeSlots,
           monthlyTimeSlots,
           startDate,
           endDate,
           lineItemUnitType,
           dayCountAvailableForBooking,
-          timeZone
+          timeZone,
+          availabilityType
+        );
+
+        const seatsOptions = getMinSeatsOptions(
+          realTimeSlots,
+          values?.bookingDates?.startDate,
+          values?.bookingDates?.endDate
```

Comment:
I assume, that this doesn't handle nightly vs daily (aka isDaily) correctly.

Reply:
A translation was missing but I think the logic calculating seats works